tango_simlib/utilities/sim_xmi_parser.py

```diff
@@ -420,9 +422,9 @@ def _get_arg_type(self, description_data):
 
         """
         if description_data.tag in ["attributes", "dynamicAttributes"]:
-            pogo_type = description_data.find("dataType").attrib.values()[0]
+            pogo_type = list(itervalues(description_data.find("dataType").attrib))[0]
```

Comment:
"Simple is better than complex." - The Zen of Python, by Tim Peters

Reply:
do you have an alternative/suggestion of how to approach to this that will be simpler?